wire-ios-sync-engine/Source/UserSession/UserSession.swift

```diff
@@ -20,9 +20,11 @@ import Foundation
 import WireDataModel
 import LocalAuthentication
 
+public typealias SelfUserType = UserType & SelfLegalHoldSubject
+
+// sourcery: AutoMockable
 /// An abstraction of the user session for use in the presentation
 /// layer.
-
 public protocol UserSession: AnyObject {
```

Comment:
Intentionally we kept this not generated by sourcery. @johnxnguyen Do you think it's time to produce the mock using Sourcery?